packages/backend/src/routers/slackRouter.ts

```diff
@@ -59,15 +60,12 @@ slackRouter.get(
     async (req, res, next) => {
         try {
             const { imageId } = req.params;
-            if (
-                !imageId.startsWith('slack-image') ||
-                !imageId.endsWith('.png')
-            ) {
+            if (!UnfurlService.isValidImageFileId(imageId)) {
                 throw new NotFoundError(
                     `Slack image not found ${req.params.imageId}`,
                 );
             }
-            const sanitizedImageId = imageId.replace('..', '');
+            const sanitizedImageId = path.normalize(imageId);
```

Comment:
Should fix

 >    .fortknight/repos/portal_lightdash/packages/backend/src/routers/slackRouter.ts